addons/mute-project/addon.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Mute project player",
+  "name": "Muted project player",
```

Comment:
Not sure about this name change.
`Ctrl+GreenFlag` could be mentioned in the addon name here maybe, similarly to 60fps

Reply:
Please don't add keybinds to addon names. That should go to descriptions.

Reply:
Why? If someone reads fast enough they might think this addon mutes all projects without needing a keybind.

Reply:
I say remove all keybinds from addon names and add the word "mode" instead.

Reply:
> Please don't add keybinds to addon names. That should go to descriptions.

Yeah, please don't. Use the description.

This is actually a nice idea. We should do this in the other addon.